superbench/benchmarks/micro_benchmarks/disk_performance.py

```diff
@@ -97,7 +100,7 @@ def add_parser_arguments(self):
                     help='Time in seconds to warm up %s test.' % io_str,
                 )
                 # Disable write tests by default
-                default_runtime = 0 if io_type == 'write' else self.__default_runtime
+                default_runtime = 0 if io_type in ['write', 'rw'] else self.__default_runtime
```

Comment:
One question: do we need to change the output data? Because there are both read and write performance.

Reply:
The output already contains both read and write data. You can check the test for this benchmark.

Reply:
Great, thanks!